firecracker-control/local.go

```diff
@@ -338,3 +338,15 @@ func (s *local) newShim(ns, vmID, containerdAddress string, shimSocket *net.Unix
 func isEADDRINUSE(err error) bool {
 	return err != nil && strings.Contains(err.Error(), "address already in use")
 }
+
+func setShimOOMScore(pid int) error {
+	score, err := sys.GetOOMScoreAdj(pid)
```

Comment:
Here we want to get the OOM score of the containerd process, but I think it's currently getting the OOM score of the shim child process instead.

This firecracker-control code runs within the containerd process itself, so to get the containerd pid all we have to do is call [`os.Getpid()`](https://golang.org/pkg/os/#Getpid). Then we can plug that into `sys.GetOOMScoreAdj`.

Reply:
Thanks for clarifying that! I thought to do the same when I was looking at containerd's implementation but got confused when I was implementing it.

Reply:
Ah okay, one more clarification. Your update is correct in that we are now using containerd's pid to get the OOM score. However, we actually want to set the OOM score on the pid you were using previously (`cmd.Process.Pid`). The end result we want is that `cmd.Process.Pid` gets the OOM score +1 of containerd's OOM score.

This is a little different than what containerd's code does, so I understand the confusion :-)